### ISSUE TITLE
Pakistan Air Quality Data

### ISSUE THREAD

**Issue by abid-omar** [openaq/openaq-fetch#421]
Government grade Air quality data is available for Lahore at http://epd.punjab.gov.pk/Air_Quality

**Comment by RocketD0g** [comment]
This is great! Thanks, @abid-omar! One item the issue will need before we can ingest the data are coordinates for each location. I didn't see these on a quick cruise of the site. Do you know where to access them or who we might ask about them? 

Another issue to flag for anyone thinking of helping write the adapter to ingest this data: The data is shared in a predictable, regular format, but in a PDF file.

**Comment by majesticio** [comment]
- Updated [site](https://epd.punjab.gov.pk/) for Environment Protection Department - Government of the Punjab
- Has some data all in PDF's, need to check against exiting data for duplicates
